README.md

```diff
@@ -1,8 +1,11 @@
 # DigitalCollections Common Java Libraries
-[![Build Status](https://travis-ci.org/dbmdz/digitalcollections-commons.svg?branch=master)](https://travis-ci.org/dbmdz/digitalcollections-commons)
-[![codecov](https://codecov.io/gh/dbmdz/digitalcollections-commons/branch/master/graph/badge.svg)](https://codecov.io/gh/dbmdz/digitalcollections-commons)
-[![codebeat badge](https://codebeat.co/badges/08e3152b-27a2-46e2-8f3d-23cd3e6e66a2)](https://codebeat.co/projects/github-com-dbmdz-digitalcollections-commons)
-[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)
+
+[![Javadocs](https://javadoc.io/badge/de.digitalcollections.commons/dc-commons.svg)](https://javadoc.io/doc/de.digitalcollections.commons/dc-commons)
+[![Build Status](https://img.shields.io/travis/dbmdz/digitalcollections-commons/master.svg)](https://travis-ci.org/dbmdz/digitalcollections-commons)
+[![Codecov](https://img.shields.io/codecov/c/github/dbmdz/digitalcollections-commons/master.svg)](https://codecov.io/gh/dbmdz/digitalcollections-commons)
+[![License](https://img.shields.io/github/license/dbmdz/digitalcollections-commons.svg)](LICENSE)
```

Comment:
Muss `LICENSE.md` heißen hier :)

Reply:
Alternativ `LICENSE.md` umbennen in `LICENSE` -> das haben wir eigentlich überall so.